packages/nimble-components/src/wafer-map/index.ts

```diff
@@ -282,6 +288,14 @@ export class WaferMap<
         return this.diesTable !== undefined;
     }
 
+    public async setData(tableData: Table): Promise<void> {
```

Comment:
If this is internal, let's add an `@internal` comment to it.

If it's part of the public API, let's add a description both in a method comment and in Storybook docs.

Same point for the `render-finished` event: if clients are expected to listen to that event, we should document it in Storybook. (We don't have a mechanism for marking events as internal other than just leaving them undocumented in storybook)

Reply:
I don't expect the render-finished event to continue to exist. I do expect setData to be public